Gemfile.lock

```diff
@@ -71,73 +71,73 @@ GEM
       jsonapi-renderer (>= 0.1.1.beta1, < 0.3)
     active_storage_base64 (2.0.0)
       rails (>= 7.0)
-    activejob (7.0.2.3)
-      activesupport (= 7.0.2.3)
+    activejob (7.0.4)
+      activesupport (= 7.0.4)
       globalid (>= 0.3.6)
-    activemodel (7.0.2.3)
-      activesupport (= 7.0.2.3)
-    activerecord (7.0.2.3)
-      activemodel (= 7.0.2.3)
-      activesupport (= 7.0.2.3)
-    activestorage (7.0.2.3)
-      actionpack (= 7.0.2.3)
-      activejob (= 7.0.2.3)
-      activerecord (= 7.0.2.3)
-      activesupport (= 7.0.2.3)
+    activemodel (7.0.4)
+      activesupport (= 7.0.4)
+    activerecord (7.0.4)
+      activemodel (= 7.0.4)
+      activesupport (= 7.0.4)
+    activestorage (7.0.4)
+      actionpack (= 7.0.4)
+      activejob (= 7.0.4)
+      activerecord (= 7.0.4)
+      activesupport (= 7.0.4)
       marcel (~> 1.0)
       mini_mime (>= 1.1.0)
-    activesupport (7.0.2.3)
+    activesupport (7.0.4)
       concurrent-ruby (~> 1.0, >= 1.0.2)
       i18n (>= 1.6, < 2)
       minitest (>= 5.1)
       tzinfo (~> 2.0)
-    acts_as_tenant (0.5.1)
+    acts_as_tenant (0.5.2)
       rails (>= 5.2)
       request_store (>= 1.0.5)
-    addressable (2.8.0)
-      public_suffix (>= 2.0.2, < 5.0)
-    airbrussh (1.4.0)
+    addressable (2.8.1)
+      public_suffix (>= 2.0.2, < 6.0)
+    airbrussh (1.4.1)
       sshkit (>= 1.6.1, != 1.7.0)
     annotate (3.2.0)
       activerecord (>= 3.2, < 8.0)
       rake (>= 10.4, < 14.0)
-    apipie-rails (0.7.0)
+    apipie-rails (0.8.2)
       actionpack (>= 5.0)
       activesupport (>= 5.0)
     ast (2.4.2)
     aws-eventstream (1.2.0)
-    aws-partitions (1.573.0)
-    aws-sdk-core (3.130.0)
+    aws-partitions (1.632.0)
+    aws-sdk-core (3.150.0)
       aws-eventstream (~> 1, >= 1.0.2)
       aws-partitions (~> 1, >= 1.525.0)
       aws-sigv4 (~> 1.1)
-      jmespath (~> 1.0)
-    aws-sdk-kms (1.55.0)
+      jmespath (~> 1, >= 1.6.1)
+    aws-sdk-kms (1.58.0)
       aws-sdk-core (~> 3, >= 3.127.0)
       aws-sigv4 (~> 1.1)
-    aws-sdk-s3 (1.113.0)
+    aws-sdk-s3 (1.114.0)
       aws-sdk-core (~> 3, >= 3.127.0)
       aws-sdk-kms (~> 1)
       aws-sigv4 (~> 1.4)
-    aws-sigv4 (1.4.0)
+    aws-sigv4 (1.5.1)
       aws-eventstream (~> 1, >= 1.0.2)
-    bcrypt (3.1.17)
+    bcrypt (3.1.18)
```

Comment:
### [bcrypt](https://github.com/bcrypt-ruby/bcrypt-ruby)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 3.1.17 → 3.1.18

[_~~change-log~~, [source-code](https://github.com/codahale/bcrypt-ruby), [gem-diff](https://my.diffend.io/gems/bcrypt/3.1.17/3.1.18)_]





<details>
<summary>Commits</summary>

A change of **10** commits. See the full changes on [the compare page](https://github.com/bcrypt-ruby/bcrypt-ruby/compare/v3.1.17...v3.1.18).

These are the individual commits:
- (49161d9) [Use `free` instead of `xfree`](https://github.com/bcrypt-ruby/bcrypt-ruby/commit/49161d9f68b90211661838f28fe2d5cf128a8b96)
- (aec1265) [Re-introduce GVL unlock](https://github.com/bcrypt-ruby/bcrypt-ruby/commit/aec1265e6c0ff8622085fd6c40fbd4d039e992ef)
- (a7e873c) [Merge pull request #260 from nwjsmith/unlock-gvl](https://github.com/bcrypt-ruby/bcrypt-ruby/commit/a7e873c7ccf59549a92157af2838d4d320428577)
- (f78b176) [Work around bug in JDK for JRuby tests](https://github.com/bcrypt-ruby/bcrypt-ruby/commit/f78b1763505d435f349a0d6dc8be18dc09418aac)
- (cc28f9e) [Update rake-compiler](https://github.com/bcrypt-ruby/bcrypt-ruby/commit/cc28f9efdc40316467e8d82095f9f8def7d0dfc2)
- (6d051de) [Fix compiler warnings](https://github.com/bcrypt-ruby/bcrypt-ruby/commit/6d051de5ac04f08cc1e681e045c582dc61555c2a)
- (87f1322) [Remove AppVeyor configuration](https://github.com/bcrypt-ruby/bcrypt-ruby/commit/87f132256456eb7cb3d403ffba245e9906bd9531)
- (e799c27) [Update for release](https://github.com/bcrypt-ruby/bcrypt-ruby/commit/e799c279b0575e87940026100c3217d0b008ab1c)
- (98f5082) [Merge pull request #261 from nwjsmith/ci-improvements](https://github.com/bcrypt-ruby/bcrypt-ruby/commit/98f5082c8cbde41b50ca8bad14263ef3b8c364e6)
- (667a136) [Merge pull request #263 from nwjsmith/three-one-eighteen](https://github.com/bcrypt-ruby/bcrypt-ruby/commit/667a136f1c4dd82ad40728300f3d579c0f26e98b)


</details>